handler/push.go

```diff
@@ -108,7 +108,7 @@ func Push(
 		}
 		if err != nil {
 			http.Error(w, err.Error(), http.StatusBadRequest)
-			level.Debug(logger).Log("msg", "failed to parse text", "err", err.Error())
+			level.Debug(logger).Log("addr", r.RemoteAddr, "msg", "failed to parse text", "err", err.Error())
```

Comment:
How about making this fully consistent with the logging in line 146, i.e. also add r.Method, and call the r.RemoteAddr entry `source` rather than `addr`?

Reply:
great!, let me check!